tests/support/util.tcl

```diff
@@ -386,6 +388,10 @@ proc createComplexDataset {r ops {opt {}}} {
             {hash} {
                 randpath {{*}$r hset $k $f $v} \
                         {{*}$r hdel $k $f}
+
+                if { [{*}$r hexists $k $f] && $usehexpire && rand() < 0.5} {
+                    {*}$r hexpire $k 1000 FIELDS 1 $f
```

Comment:
can this be put into randpath?
i doubt why we need the `usehexpire` parameter?

Reply:
blindly followed `useexpire`. Removed. thanks.